www/static/js/docs.js

```diff
@@ -40,7 +40,7 @@ $(document).ready(function () {
 
     // Table of Contents
     $('#page-toc').toc({
-        'selectors':         'h1,h2,h3,h4,h5,h6', // elements to use as headings
+        'selectors':         'h1,h2', // elements to use as headings
```

Comment:
I guess that's an ok workaround. But doesn't that mean that we should use different tags other than headers in the references? This might become confusing when we do use `h3`s properly in a document and then it won't show up in the ToC.